_clients/logstash/read-from-opensearch.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Read from OpenSearch
+parent: Logstash
+nav_order: 220
+---
+
+# Read from OpenSearch
+
+As we ship Logstash events to an OpenSearch cluster using the OpenSearch output plugin, we can also perform read operations on an OpenSearch cluster and load data into Logstash using the [OpenSearch input plugin](https://github.com/opensearch-project/logstash-input-opensearch).
+
+The OpenSearch input plugin reads the search query results performed on an OpenSearch cluster and loads them into Logstash. This lets you replay test logs, reindex, and perform other operations based on the loaded data. You can schedule ingestions to run periodically by using 
+[cron expressions](https://opensearch.org/docs/latest/monitoring-plugins/alerting/cron/), or manually load data into Logstash by running the query once.
+
+
+
+## OpenSearch input plugin
+
+To run the OpenSearch input plugin, add the configuration to the `pipeline.conf` file within your Logstash's `config` folder. The example below runs the `match_all` query filter and loads in data once.
+
+```yml
+input {
+  opensearch {
+    hosts       => "https://hostname:port"
+    user        => "admin"
+    password    => "admin"
+    index       => "logstash-logs-%{+YYYY.MM.dd}"
+    query       => "{ "query": { "match_all": {}} }"
+  }
+}
+
+filter {
+}
+
+output {
+}
+```
+
+To ingest data according to a schedule, use a cron expression that specifies the schedule you want. 
+For example, to load in data every minute, add `schedule => "* * * * *"` to the input section of your `pipeline.conf` file.
```

Comment:
The second sentence here doesn't need to be on a new line.

Reply:
ack